hdt-lib/include/HDTVocabulary.hpp

```diff
@@ -33,12 +33,13 @@
 #define HDT_HDTVOCABULARY_HPP_
 
 #include <string>
+#include <HDTVersion.hpp>
 
 namespace hdt {
 namespace HDTVocabulary {
 	// Base
 	const std::string HDT_BASE = "<http://purl.org/HDT/hdt#";
-	const std::string HDT_CONTAINER = HDT_BASE+"HDTv1>";
+	const std::string HDT_CONTAINER = HDT_BASE+"HDTv"+HDT_VERSION+">";
```

Comment:
Would it make sense to include index and release in here? (Even though only the main version is supposed to induce breaking changes, certain releases might introduce bugs in generated HDT files, as we had before. Might be good to be able to detect them.)